packages/cf-component-button/package.json

```diff
@@ -10,7 +10,8 @@
   },
   "dependencies": {
     "cf-style-container": "^0.0.2",
-    "react": "^0.14.2 || ^15.0.0-0"
+    "css-color-function": "^1.3.0",
+    "react": "^15.0.0-0"
```

Comment:
is it intentional to drop support of 0.14? I think support of last 2 major versions is a good strategy.